blocks/article-body-block/chains/article-body/default.jsx

```diff
@@ -44,84 +49,66 @@ function parseArticleItem(item, index, arcSite, phrases, id, customFields) {
 	switch (type) {
 		case "text": {
 			return content && content.length > 0 ? (
-				<Paragraph key={key} dangerouslySetInnerHTML={{ __html: content }} />
+				<Paragraph key={`${type}_${key}`} dangerouslySetInnerHTML={{ __html: content }} />
 			) : null;
 		}
 		case "copyright": {
 			return content && content.length > 0 ? (
 				<Paragraph
-					key={key}
+					key={`${type}_${key}`}
 					className={`${BLOCK_CLASS_NAME}__copyright`}
 					dangerouslySetInnerHTML={{ __html: content }}
 				/>
 			) : null;
 		}
 
 		case "divider": {
-			return <hr className={`${BLOCK_CLASS_NAME}__divider`} key={key} />;
+			return <hr className={`${BLOCK_CLASS_NAME}__divider`} key={`${type}_${key}`} />;
 		}
 
 		case "image": {
 			const {
-				url,
-				subtitle,
+				additional_properties: { link = "" } = {},
+				// alignment not always present
+				alignment = "",
+				alt_text: altText,
+				auth,
 				caption,
 				credits,
-				alt_text: altText,
+				subtitle,
+				url,
 				vanity_credits: vanityCredits,
-				// alignment not always present
-				alignment = "",
-				additional_properties: additionalProperties = {},
 			} = item;
 
-			// link url set in composer
-			const { link = "" } = additionalProperties;
-
 			// only left and right float supported
 			const allowedFloatValue = alignment === "left" || alignment === "right" ? alignment : "";
-
-			let figureImageClassName = `${BLOCK_CLASS_NAME}__image`;
-
-			if (allowedFloatValue) {
-				// add space after initial string ' '
-				figureImageClassName +=
-					allowedFloatValue === "left"
-						? ` ${BLOCK_CLASS_NAME}__image-float-left`
-						: ` ${BLOCK_CLASS_NAME}__image-float-right`;
-			}
+			const figureImageClassName = `${BLOCK_CLASS_NAME}__image${
+				allowedFloatValue ? ` ${BLOCK_CLASS_NAME}__image-float-${allowedFloatValue}` : ""
+			}`;
 
 			if (url) {
-				const ArticleBodyImage = () => <Image src={url} alt={altText} />;
 				const formattedCredits = formatCredits(vanityCredits || credits);
-
-				const ArticleBodyImageContainer = ({ children }) => (
+				return (
 					<MediaItem
-						key={key}
+						key={`${type}_${key}`}
 						className={figureImageClassName}
 						caption={!hideImageCaption ? caption : null}
 						credit={!hideImageCredits ? formattedCredits : null}
 						title={!hideImageTitle ? subtitle : null}
 					>
-						{children}
+						<Conditional component={Link} condition={link} href={link}>
+							<Image
+								src={imageANSToImageSrc(item)}
+								alt={altText}
+								resizerURL={RESIZER_URL}
+								resizedOptions={{ auth: auth[RESIZER_APP_VERSION] }}
+								width={item.width}
```

Comment:
Do we want to use the raw `width` and `height` here? For the testing article I am seeing 3024x4032

Reply:
I wanted to keep the aspect ratio and I suppose I could do some math to reduce it to something more in line, but with the responsive resizing it could be any value in a range.  I could probably figure out the max width (which will be different depending on if it's floating...) and then calculate the height.  If I did this though, we'd want to be able to do this anywhere image is used and it's getting to the point where we want yet another util...

Reply:
I broke out a method in this to hopefully manage this a little better.